widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -361,6 +365,12 @@ public void initChat(String companyName,
         createNewTimerCallback();
         callTimer.addFormattedValueListener(timerStatusListener);
         updateAllowFileSendState();
+
+        // TODO: enable chat panel with send message support
+        // will implement in the next task
+        if (isSecureEngagementUseCase.invoke()) {
+            emitViewState(chatState.enableChatPanel());
+        }
```

Comment:
To double-check, is this TODO still relevant? I'm asking because it looks like `enableChatPanel()` does what TODO states.

Reply:
Yes, it just show add attachment button. Needs to change.